lib/observation_search.rb

```diff
@@ -29,20 +29,14 @@ def site_search_params(site, params = {})
 
     def search_in_batches(raw_params, options={}, &block)
       search_params = Observation.get_search_params(raw_params, options)
-      search_params.merge!({ page: 1, per_page: 500, preload: [ ],
+      search_params.merge!({ min_id: 1, limit: 500, preload: [ ],
         order_by: "id", order: "asc" })
-      scope = Observation.get_search_scope_or_elastic_results(search_params)
-      if scope.is_a?(ActiveRecord::Relation)
```

Comment:
Since I pushed the `min_id` and `limit` handling down into both the db-based and es-based query implementations, `Observation.get_search_scope_or_elastic_results(search_params)` should handle deciding which kind of query to use and both implementations should use `min_id` and `limit` correctly. Both return objects that can be used to get the ID of the last result, so it seemed simpler to remove the conditional.